src/tinuous/__main__.py

```diff
@@ -152,6 +152,7 @@ def fetch(cfg: Config, state: str, sanitize_secrets: bool) -> None:
             msg += f", {artifacts_added} artifacts added"
         if relassets_added:
             msg += f", {relassets_added} release assets added"
+        msg += f"\n\nProduced by tinuous v{__version__}"
```

Comment:
since we do not have `v` prefix in the tags or release versions, I do not see a need to prepend `v` here, why not to keep it consistent with output of `tinuous --version` and thus simply

```suggestion
        msg += f"\n\nProduced by tinuous {__version__}"
```
?

Reply:
I just think "Produced by tinuous v0.3.0" looks better than "Produced by tinuous 0.3.0".  No one's going to expect that the version here will be exactly the same as the tag name.

Reply:
oh well, I don't care about looks enough I guess, and more about consistency (not necessarily to match the tags but overall, and matching to the tags just becomes a side-effect of that). Do you feel strong about the "looks"? ;-)

Reply:
No, I don't feel strongly, I was just stating my case.  I'll get rid of the `v`.